btx/diagnostics/run.py

```diff
@@ -1,14 +1,17 @@
 import numpy as np
 import argparse
 import os
+from mpi4py import MPI
 import matplotlib.pyplot as plt
 from mpl_toolkits.axes_grid1 import make_axes_locatable
 import matplotlib.colors as colors
 from matplotlib.colors import LogNorm
+from btx.interfaces.ipsana import *
 
 from psana import EventId
-from btx.interfaces.ipsana import *
-from mpi4py import MPI
+from Detector.UtilsEpix10ka import find_gain_mode
+from Detector.UtilsEpix10ka import info_pixel_gain_mode_statistics_for_raw
+from Detector.UtilsEpix10ka import map_pixel_gain_mode_for_raw
```

Comment:
Shouldn't we load these only if the detector is epix10k?

Reply:
Good catch. Note to future self: relevant functions for Jungfrau4M are here: https://github.com/lcls-psana/Detector/blob/master/src/UtilsJungfrauCalib.py.

Reply:
Also described in Issue #161.